exercises/D-strings/exercise.js

```diff
@@ -1,3 +1,8 @@
 // Start by creating a variable `message`
 
+var message = "This is a string";
+var messageType = typeof message;
 console.log(message);
+console.log(messageType);
```

Comment:
You can write this using **\n** which allows to use once **console.log()**. Have a go.